cub/test/catch2_test_device_transform.cu

```diff
@@ -378,6 +379,21 @@ C2H_TEST("DeviceTransform::Transform fancy output iterator type", "[device][devi
   REQUIRE(result == c2h::device_vector<type>(num_items, (13 + 35) + 4));
 }
 
+C2H_TEST("DeviceTransform::Transform fancy output iterator type with void value type", "[device][device_transform]")
+{
+  using type          = int;
+  const int num_items = GENERATE(100, 100'000); // try to hit the small and full tile code paths
+  c2h::device_vector<type> a(num_items, 1);
+  c2h::device_vector<type> b(num_items, 2);
+  c2h::device_vector<type> result(num_items, thrust::no_init);
+
+  using It = cub::CacheModifiedOutputIterator<cub::CacheStoreModifier::STORE_DEFAULT, int>;
```

Comment:
suggestion: we use pascal case only for template parameters. Let's use snake case here.